src/dashboard/src/templates/transfer/detail.html

```diff
@@ -18,14 +18,14 @@
         <div class="alert-message block-message">
           <p>There are micro-services awaiting user decisions.</p>
           <div class="alert-actions">
-            <a class="btn small" href="{% url 'components.ingest.views.ingest_microservices' uuid %}">View micro-services</a>
```

Comment:
Yup, good catch.  Though, the views are almost identical except for the breadcrumbs.